docs/pbc/all/business-intelligence/202410.0/amazon-quicksight-third-party-integration/best-practices-analyzing-shop-data.md

```diff
@@ -0,0 +1,158 @@
+3. Click **Select**
+  This shows a success message and the **Datasets in this analysis** window. Click **Close** to close the window.
+4. In the **Visuals** pane, click **+ADD**.
+This adds an empty visual to the sheet.
+5. Click on the visual you've added.
+6. In the **Data** pane, for **Dataset**, select **Category + Localized Product Abstract**.
+7. Add the **name[spy_category_attribute]** as a dimension to the **Y AXIS** section.
+8. Add the **sku** field as a measure to the **VALUE** field.
+
+![product-per-category](https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/business-intelligence/amazon-quicksight-third-party-integration/best-practices-analyzing-shop-data.md/product-per-category.png)
+
+9. Name of the visual is inherited from the fields you've added. To make it presentable, double-click the name.
+10. In the **Edit title** window, enter a name for the visual. For example, `Products per category`.
+11. To apply the change, click **Save**.
+
+This closes the the window and the new name of the visual is displayed.
+
+Now you know how to add data and format it in different ways. For more information on how to create analyses, see [Author Workshop](https://catalog.workshops.aws/quicksight/en-US/author-workshop).
+
+
+## Add filters
+
+1. In the toolbar, click the filter button.
+
+2. In the **Filters** pane, click **+ ADD** and select **store**.
+  This adds the filter to the pane.
+3. To edit the filter, click on it.
+4. To apply the filter to all the visual in the sheet, for **Applied to**, select **Single sheet**.
+5. To add filter controls, click the three dots next to the filter and select **Add control**>**Top of this sheet**.
+  This adds the filter controls to the top of the sheet.
+
+![store-filter](https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/business-intelligence/amazon-quicksight-third-party-integration/best-practices-analyzing-shop-data.md/store-filter.png)  
+
+
+## Share the analysis
+
+If you want another author to collaborate with you on the analysis, you can share it.
+
+To share an analysis, follow the steps:
+
+1. On the page of the analysis, go to **File** > **Share**.
+2. In the **Share analysis**, enter and select the email address of the author to share the analysis with.
+  This displays the user in the table.
+3. Repeat the previous step until you've added all the authors you want to share the analysis with.
+4. Click **Share**
+  This closes the window. The added users can now edit the analysis.
+
+{% info_block infoBox "Manage permissions" %}
+
+You can check and manage permissions of all users to an analysis in **File**>**Share**>**Manage analysis permissions**.
+
+{% endinfo_block %}
+
+
+
+
+## Publish analysis as a dashboard and share
+
+Most of your analytics users will have view permissions. When you're ready to share the analysis with a wider audience, you need to publish it as a dashboard. Users with view permissions can access only dashboards.
```

Comment:
```suggestion
Most of your analytics users will be readers and have view permissions for dashboards. When you're ready to share the analysis with a wider audience, you must publish it as a dashboard so that the readers can access it.
```